addons/portal/static/src/js/portal_sidebar.js

```diff
@@ -30,9 +31,9 @@ var PortalSidebar = publicWidget.Widget.extend({
     _setDelayLabel: function () {
         var $sidebarTimeago = this.$el.find('.o_portal_sidebar_timeago').toArray();
         $sidebarTimeago.forEach((el) => {
-            var dateTime = moment(time.auto_str_to_date($(el).attr('datetime'))),
-                today = moment().startOf('day'),
-                diff = dateTime.diff(today, 'days', true),
+            var dateTime = DateTime.fromJSDate(time.auto_str_to_date($(el).attr('datetime'))),
```

Comment:
This could probably work:
```suggestion
            var dateTime = deserializeDateTime($(el).attr('datetime')),
```

Maybe the string coming from `$(el).attr('datetime')` needs a bit of adaptation but I don't think so as the datetime attr directly comes from the server.

See portal_my_invoice_detail